ArduPlane/tailsitter.cpp

```diff
@@ -334,7 +334,8 @@ void QuadPlane::tailsitter_speed_scaling(void)
         if (is_zero(throttle)) {
             spd_scaler = tailsitter.throttle_scale_max;
         } else {
-            spd_scaler = constrain_float(hover_throttle / throttle, 1.0f, tailsitter.throttle_scale_max);
+            spd_scaler = constrain_float(hover_throttle / throttle, tailsitter.gain_scaling_min, tailsitter.throttle_scale_max);
```

Comment:
this is inside an if ((spd_scaler >= 1.0f) && ...
so this change is non-functional/unnecessary

Reply:
We will hit this of the the first method is not enabled

Reply:
I was assuming that tailsitter.gain_scaling_min is < 1
Is that not guaranteed?

Reply:
Yeah: 0 < gain_scaling_min  <  1. It must reduce throttle, it does on stable. https://github.com/ArduPilot/ardupilot/blob/f0d8ca5e371b7f8b1c1b883fed87043d98d2cfe3/ArduPlane/tailsitter.cpp#L254

Scaling to less than 1 is required to get it working well at high throttle, especially on vectored.

Reply:
OK, that change came in with commit 95da094, in April 2019.
Probably wasn't tested adequately on TVBS; IIRC we didn't think this option was very useful at the time.
But this means the bitmask value name is misleading as are the comments which say "boost gains at low throttle"

Reply:
We also seems to have lost all scaling of thrust vectoring in the same commit.

Reply:
I believe the reasoning was that thrust vectoring would not be dependent on airspeed in the same way as control surfaces.

Reply:
We should use throttle based scaling on them all the time, I will update this PR.